apps/prairielearn/src/pages/partials/navbar.ejs

```diff
@@ -345,6 +345,12 @@
   </div>
 </nav>
 
+<% if (typeof course !== 'undefined' && course.banner_html && course.banner_color) { %>
+  <div class="alert alert-<%= course.banner_color %> mb-0 rounded-0 text-center">
+    <%- course.banner_html %>
+  </div>
+<% } %>
```

Comment:
This currently shows on all course pages. We could in theory use the value of `navPage` to limit this to instructor-facing pages?

Reply:
That's probably a good idea. That way instructors can continue to use the existing CA course instances to run makeup exams at the start of the new semester, without confusing students with a "this course has moved" messages.

Reply:
Done in da38600.